docs/source/python/dataset.rst

```diff
@@ -570,7 +570,7 @@ dataset to be partitioned.  For example:
     part = ds.partitioning(
         pa.schema([("c", pa.int16())]), flavor="hive"
     )
-    ds.write_dataset(table, "sample_dataset", format="parquet", partitioning=part)
+    ds.write_dataset(table, "partitioned_dataset", format="parquet", partitioning=part)
```

Comment:
We already wrote to "sample_dataset" a bit above, so this now started to (correctly) raise an error if using the same name